src/lib.rs

```diff
@@ -1,31 +1,46 @@
 /*!
 # Unicode Blocks
 
-This crate contains a list of all unicode blocks and provides some functions to search across them.
+This crate contains a list of all unicode blocks and provides some functions
+to search across them.
+
```

Comment:
I don't wrap comments.

Reply:
Ah, sorry about that. That was habitual. :)